### PR TITLE
WIP: Chore: add t3b1 support suite

### PR DESCRIPTION
**Pull request by karliatto**
## Description
This is adding T3B1 support to suite. It is not completed, there are missing parts:

To have full support we will need:
- [ ]  support of coins to include T3B1, @matejcik https://github.com/trezor/trezor-firmware/blob/main/common/defs/support.json
- [ ] Add firmware in packages/connect-common/files/firmware/t3b1
-

### CODE REVIEW

File: packages/suite/src/middlewares/suite/suiteMiddleware.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ const suite =
         }
 
         if (deviceActions.connectDevice.match(action)) {
+            // TODO: should this apply also for DeviceModelInternal.T3B1 ???
```
Comment (karliatto) [inline packages/suite/src/middlewares/suite/suiteMiddleware.ts:74]: I would expect T3T1 to have similar feature set that T3B1 but I am not sure.

Comment (MiroslavProchazka) [reply packages/suite/src/middlewares/suite/suiteMiddleware.ts:74]: You mean similar to T2B1, right?

**Review by mroz22 — CHANGES_REQUESTED** [review]
carlos, could you please try to make these places type-safe? in the meaning that unless all internal models are defined, typecheck will catch?

**Review by komret — CHANGES_REQUESTED** [review]
T3B1 should mimic T2B1, not T3T1. It is the same device as T2B1, but with a different chip.

**Comment by mroz22** [comment]
@komret doesn't your team want to take this from @karliatto? it looks like all the changes are to be done on the suite level mostly.

**Comment by MiroslavProchazka** [comment]
> @komret doesn't your team want to take this from @karliatto? it looks like all the changes are to be done on the suite level mostly.

Yes, we will take over. Komár is already working on it.

**Comment by komret** [comment]
Closing in favour of https://github.com/trezor/trezor-suite/pull/13544

**Pull request closed without merging**